src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -163,59 +163,54 @@ if (__DEV__) {
   var debugCounter = 1;
 }
 
-// This is a constructor of a POJO instead of a constructor function for a few
-// reasons:
+function FiberNode(tag: TypeOfWork, key: null | string) {
```

Comment:
I attempted to do this, but it confuses Flow as I cannot export the class (it won't allow me to do so)?